### PR TITLE
Feat : refactored FC VAT tests

### PR DESCRIPTION
**Pull request by thadeusssewanyana**

### CODE REVIEW

File: src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature [hunk 1]
```diff
@@ -2,175 +2,153 @@ Feature: FC VAT Debt Calculation End point testing
...
-      | 123    | 500000         | 2018-12-16 | 2019-04-14          | Y                 |
+      | debtItemChargeId  | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
+      | debtItemChargeId1 | 500000         | 2018-12-16 | 2022-01-04          | Y                 |
     And the fc vat debt item has payment history
       | paymentAmount | paymentDate |
       | 100000        | 2019-02-03  |
-    And the fc vat customer has breathing spaces applied
-      | debtRespiteFrom | debtRespiteTo |
-      | 2020-04-06      | 2021-04-06    |
+    And no breathing spaces have been applied to the fc vat customer
```
Comment (vinniebrice) [inline src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature:10]: For the bug https://jira.tools.tax.service.gov.uk/browse/DTD-1023 this is great! Although BS tests will need to be added later

File: src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature [hunk 2]
```diff
@@ -2,175 +2,153 @@ Feature: FC VAT Debt Calculation End point testing
...
     When the debt item is sent to the fc vat ifs service
     Then the fc vat ifs service wilL return a total debts summary of
       | combinedDailyAccrual | unpaidAmountTotal |
       | 0                    | 400000            |
-#    Uncomment and fix step below when DTD-1023 has been fixed
-#    And the 1st fc vat debt summary will contain
-#      | debtId | interestDailyAccrual | interestRate |
-#      | 123    | 28                   | 2.75         |
+    And the 1st fc vat debt summary will contain
```
Comment (vinniebrice) [inline src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature:32]: Atm this bug https://jira.tools.tax.service.gov.uk/browse/DTD-1022 is not in development and therefore these tests will most likely fail. Would it be better to add these in a separate pr for the other bug?

File: src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature (hunk 1 above)
Comment (thadeusssewanyana) [reply src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature:10]: Yes @vinniebrice this is for the bug

File: src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature (hunk 2 above)
Comment (thadeusssewanyana) [reply src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature:32]: @vinniebrice they're passing based on the IFS DTD-871 branch looks like DTD-1022 might not be needed

Comment (vinniebrice) [reply src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature:32]: Awesome!

**Review by vinniebrice — APPROVED** [review]
